src/app/authentication/authentication-page/register-page/register-form/register-form.component.ts

```diff
@@ -0,0 +1,40 @@
+import { Component, OnInit } from '@angular/core';
+import { FormBuilder, Validators } from '@angular/forms';
+import { Router } from "@angular/router";
+
+import { AuthenticationService } from '../../../../core/services/authentication-service/authentication.service'
+
+@Component({
+  selector: 'app-register-form',
+  templateUrl: './register-form.component.html',
+  styleUrls: ['./register-form.component.scss']
+})
+export class RegisterFormComponent implements OnInit {
+  errorMessage: string;
+ 
+  registerForm = this.fb.group({
+    email: ['', [Validators.required, Validators.email]],
+    password: ['', [Validators.required, Validators.minLength(6)]]
+  });
+
+  constructor(
+    public auth: AuthenticationService,
+    private fb: FormBuilder,
+    private router: Router
+  ) { }
+
+  ngOnInit() {
+  }
+
+  submit(value){
+    this.auth.doRegister(value)
```

Comment:
this code is similar to the one in forgot-pass-form.component.ts. Why don't you try to reuse it?
It's a good approach to follow the DRY principle.